### DISCUSSION TITLE
Retry Button, when Printer is busy

### DISCUSSION THREAD

**Discussion by aeneasw** [LycheeOrg/Lychee#2786] [Ideas]
### Enhancement

In 25% of all cases when I send a print wireles to my Elegoo Saturn 3 ultra after uploading it fails with "Printer is Busy".

Rerendering takes often a lot of time, and to resend it, I have to rerender it - right?
Can't there be an additional Button "Try again" on the "Printer is Busy" Alert, where I can only try to send it again without the need of rerendering the whole scene?

**Comment by d7415** [comment]
I think you're in the wrong place
